cmd/cryptcheck/cryptcheck.go

```diff
@@ -80,6 +80,8 @@ func cryptCheck(ctx context.Context, fdst, fsrc fs.Fs) error {
 	}
 	defer close()
 
+	opt.ReportDirs = check.ReportDirs
```

Comment:
We shouldn't need this - it should be in the return of `check.GetCheckOpt` already.

Reply:
I could have sworn there was a reason for needing to do this... but 10 months later I can't remember what it was! I guess I'll try removing it and see if anything breaks 😅